src/PlaneWaveBasis.jl

```diff
@@ -331,6 +331,11 @@ end
 The wave vectors `G` in reduced (integer) coordinates for a cubic basis set
 of given sizes.
 """
+function G_vectors(fft_size::Union{Tuple,AbstractVector}, array_type::UnionAll)
```

Comment:
UnionAll is an implementation detail I think. Type it as ::Type if you really want to type it (ie if you need it for dispatch or for clarity) or just don't type it at all

Reply:
I think I'm going to change it to ::Type just for code clarity purpose.